test/extended/router/config_manager.go

```diff
@@ -141,6 +141,7 @@ func waitForRouteToRespond(ns, execPodName, proto, host, abspath, ipaddr string,
 		set -e
 		STOP=$(($(date '+%%s') + %d))
 		while [ $(date '+%%s') -lt $STOP ]; do
+			rc=0
 			code=$( curl -k -s -m 5 -o /dev/null -w '%%{http_code}\n' --resolve %s:%d:%s %q ) || rc=$?
 			if [[ "${rc:-0}" -eq 0 ]]; then
```

Comment:
It might make sense to replace `${rc:-0}` with `$rc` now that we know `rc` is always initialized, but using `${rc:-0}` doesn't actually hurt anything.